OrbitCore/OrbitModule.cpp

```diff
@@ -48,8 +48,7 @@ std::string Module::GetPrettyName() {
 
 //-----------------------------------------------------------------------------
 bool Module::IsDll() const {
-  return ToLower(Path::GetExtension(m_FullName)) ==
-             std::string(".dll") ||
+  return ToLower(Path::GetExtension(m_FullName)) == std::string(".dll") ||
          Contains(m_Name, ".so");
```

Comment:
absl::EndsWith()?

Reply:
This change is actually just because I ran clang-format. I would change things like that in a different PR